Rlabkey/R/labkey.domain.R

```diff
@@ -241,7 +244,7 @@ labkey.domain.createAndLoad <- function(baseUrl=NULL, folderPath, name, descript
     labkey.domain.create(baseUrl = baseUrl, folderPath = folderPath, domainKind = domainKind,
         domainDesign = design, options = options)
 
-    labkey.insertRows(baseUrl = baseUrl, folderPath = folderPath, schemaName = schemaName, queryName= name, df)
+    labkey.query.import(baseUrl, folderPath, schemaName, queryName= name, toImport = df)
```

Comment:
@labkey-klum What do you think of this change? The new `labkey.query.import` call is much faster then the `labkey.insertRows` or `labkey.importRows` calls. Do you recall any specific reason why the `labkey.domain.createAndLoad` would need to use the `insertRows` or do you think it is fine to change it to `labkey.query.import`?

Reply:
yes, agreed that changing this to use the new `query.import` function will improve the perf on this function.